docs/courses/intro_to_gui.rst

```diff
@@ -51,4 +51,213 @@ or in Python
         ``cd path_to_your_Desktop/course/intro_to_gui``
 
 2. 
-    MORE THINGS HERE
\ No newline at end of file
+    **Launch python**
+
+        ``python``
+
+3. 
+    **Make Python load NEURON's h and gui models by entering this command at the >>> prompt:**
+
+.. code::
+    python
+
+    from neuron import h. gui
+
+4. 
+    **Get a CellBuilder**
+
+        :menuselection:`Main Menu --> Build --> CellBuilder`
+
+        .. image:: img/cellbuilder.png
+            :align: center
+
+5. **Specify surface area**
+
+:menuselection:`CellBuilder --> geometry`
+
+Make sure there is a check mark in the "Specify Strategy" box.
+
+Select "area".
+
+Click on "Specify Strategy" to clear the check mark.
+
+Make sure that area is 100 um^2. 
```

Comment:
I wonder if it's better to say "um" because that's what NEURON says, or to say "µm" since that's the right name of the unit.